contracts/src/challengeV2/WhitelistedEdgeChallengeManager.sol

```diff
@@ -0,0 +1,43 @@
+// Copyright 2023, Offchain Labs, Inc.
+// For license information, see https://github.com/offchainlabs/bold/blob/main/LICENSE
+// SPDX-License-Identifier: BUSL-1.1
+//
+pragma solidity ^0.8.17;
+
+import "./EdgeChallengeManager.sol";
+import "../rollup/IRollupCore.sol";
+
+interface IWhitelistedEdgeChallengeManager is IEdgeChallengeManager {
+    function accountHasMadeRival(bytes32 mutualId, address account) external view returns (bool);
+}
+
+contract WhitelistedEdgeChallengeManager is EdgeChallengeManager, IWhitelistedEdgeChallengeManager {
```

Comment:
if we are inheriting `EdgeChallengeManager`, will might want to leave a gap in `EdgeChallengeManager` for upgradabiltiy